src/components/date/date.jsx

```diff
@@ -0,0 +1,249 @@
+import PropTypes from 'prop-types';
+import React, {
+  forwardRef,
+  useEffect,
+  useLayoutEffect,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from 'react';
+import calendarSvg from '../../svgs/calendar.svg';
+import cautionSvg from '../../svgs/caution.svg';
+import Calendar from '../calendar/calendar.jsx';
+import FormControl from '../form-control/form-control.jsx';
+import IconButton from '../icon-button/icon-button.jsx';
+import Icon from '../icon/icon.jsx';
+import useDropdownClose from '../../hooks/use-dropdown-close.js';
+import styles from './date.css';
+
+/* eslint-disable react/prop-types */
+function ControlIcons(props) {
```

Comment:
Given that this is a "private" component, I opted to avoid defining a prop-type interface for now. If we do move it to its own file, that would be a good time to add a prop-types interface and test it.

Reply:
This private component can be extracted to share logic / presentation about icons in form controls.

Reply:
We extracted this work and styling into a new `FormControlIcons` component. We can talk about/review that here: https://github.com/mavenlink/design-system/pull/296